packages/cli/build/main.ts

```diff
@@ -35,9 +38,12 @@ async function main(mode) {
     args.filter((x) => !x.startsWith('-')).forEach((target) => {
         command.push('--env', target)
         if (!knownTargets.includes(target)) {
-            throw new TypeError('Unknown target ' + target + '. Known targets: ' + knownTargets.join(','))
+            throw new TypeError(`Unknown target ${target}. Known targets: ${knownTargets}`)
         }
     })
-    spawn('npx', ['webpack', ...command], { stdio: 'inherit', shell: true, cwd: resolve(__dirname, '../maskbook/') })
+    return spawnSync('npx', ['webpack', ...command], {
```

Comment:
The child_process.spawnSync() method is generally identical to child_process.spawn() with the exception that the function will not return until the child process has fully closed.

I think this is not correct. It will block other daemons to do their tasks

Reply:
OK, i need handle exit code logic,
if non-zero them exit program